clang/test/SemaTemplate/instantiate-pure-virtual-function.cpp

```diff
@@ -0,0 +1,20 @@
+// RUN: %clang_cc1 -fsyntax-only -verify -Wundefined-func-template %s
+// expected-no-diagnostics
+
+namespace PR74016 {
```

Comment:
I think we normally call them GH when a bug was filed as a GitHub issue.
```suggestion
namespace GH74016 {
```

Reply:
Fixed. I wasn't aware of this, thanks!